Makefile

```diff
@@ -172,7 +172,7 @@ lint: agentlint
 # We have to run test twice: once for all packages with -race and then once
 # more without -race for packages that have known race detection issues.
 test:
-	$(GO_ENV) go test $(GO_FLAGS) -race ./...
+	$(GO_ENV) go test $(GO_FLAGS) -race $(shell go list ./... | grep -v /integration-tests/)
```

Comment:
IMO `make test` should run all the tests to help me diagnose a failing PR check? 

We could have a `make integration-test` target if the above is not a good idea. The only thing is that I'd like the CI job to clearly state what to run locally to repro it.

Reply:
make test is used in drone pipelines where the integration tests cannot run. I will create a make integration-test for now